### ISSUE TITLE
nixpkgs-mozilla does not evaluate on unstable and 18.09

### ISSUE THREAD

**Issue by Mic92** [mozilla/nixpkgs-mozilla#110]
```
$ nix-env -f package-set.nix -qa '*' --meta --xml --drv-path --show-trace --arg pkgs "import <nixpkgs> {}" >/dev/null
error: while querying the derivation named 'servo-latest':
while evaluating the attribute 'buildInputs' of the derivation 'servo-latest' at /home/joerg/git/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:179:11:
while evaluating 'getOutput' at /home/joerg/git/nixpkgs/lib/attrsets.nix:452:23, called from undefined position:
while evaluating anonymous function at /home/joerg/git/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:127:17, called from undefined position:
attribute 'mesa' missing, at /home/joerg/git/nixpkgs-mozilla/pkgs/servo/default.nix:7:4
```

**Comment by garbas** [comment]
@Mic92 feel free to remove servo

**Comment by nbp** [comment]
This seems to be fixed by #114.
